.travis.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# https://github.com/travis-ci/travis-rubies
```

Comment:
Please add a comment about `()` vs `{}` here :-)

```
Some functions are declared using `func_name() (...)` and some  `func_name() { }`. This is because we need ...
```

Otherwise I know I will change such stuff in a few month when I don't remember it...